docs/Documentation/Resource_Management_System/index.md

```diff
@@ -0,0 +1,55 @@
+# Lex Allocation Management System (Beta)
+*Lex is a system for managing your high-performance computing (HPC) resource assignments, account information, and tracking utilization.*
+
+*Note:  Lex is currently in development with beta functionality available.*
+
+## Access
+
+An [NREL HPC account](https://www.nrel.gov/hpc/user-accounts.html) is required to access Lex. To login to Lex, open a web browser to [https://hpcprojects.nrel.gov](https://hpcprojects.nrel.gov/login/?next=/). Login with your NREL HPC username and password. An OTP token is not required to authenticate. 
+
+## Managing Users
+
+The allocation PI, Project Lead, and Alternate Project Lead can use Lex to manage the allocation entitlement list. 
+
+To add and remove users, navigate to the "Manage Users" page for the allocation.
+![image](../../assets/images/Lex/lex-manage-users-nav.png)
+
+To add users, click the "Add User" button and enter the user's email. Repeat for any additional users. Click "Delete" to remove users from the allocation entitlement list.
+**Be sure to click "Submit Changes" to apply your changes.**
+
+![image](../../assets/images/Lex/lex-manage-users.png)
+
+
+Users that have an existing NREL HPC account will be added to the allocation group and permissions will be granted within 24 hours. 
```

Comment:
"Users who..." (because the user represents a person, not a thing). And for known users it should be nearly immediate now, right?